src/experiment/metrics/basicmetrics.jl

```diff
@@ -94,8 +113,19 @@ function (metrics::BasicMetrics{TakeObjective, <:LearnedHeuristic})(model::CPMod
     index = findfirst(!isnothing, model.statistics.solutions) #return the list of index of real solution in model.statistics.solutions
     push!(metrics.meanNodeVisitedUntilfirstSolFound, !isnothing(index) ? model.statistics.nodevisitedpersolution[index] : nothing)
     push!(metrics.timeneeded,dt)
-    push!(metrics.scores,copy(model.statistics.objectives))
-    push!(metrics.totalReward,last_episode_total_reward(metrics.heuristic.agent.trajectory))
+    if !isempty(model.statistics.objectives)
```

Comment:
Did it cause an error to push an empty array? If not, I don't think this change is desirable as it could cause problems in the `SeaPearlExtras` plotting functions.

Reply:
I did this because of an error (despite being aware that it might cause conflicts with `SeaPearlExtras` but now I don't get this error anymore when running graphcoloring so I removed it but we should be ready to reverse this change if we see an error related to this line in the future.